src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -68,6 +72,18 @@ def get_attrs(self, item_list, user, **kwargs):
                 type = actor_type_to_string(alert_rule.owner.type)
                 result[alert_rule]["owner"] = f"{type}:{resolved_actors[type][alert_rule.owner_id]}"
 
+        if "original_alert_rule" in self.expand:
+            for alert_rule in alert_rules.values():
+                if alert_rule.status == AlertRuleStatus.SNAPSHOT.value:
+                    snapshot_activity = AlertRuleActivity.objects.filter(
+                        alert_rule=alert_rule,
+                        type=AlertRuleActivityType.SNAPSHOT.value,
+                    )
+                    if snapshot_activity:
+                        result[alert_rule][
+                            "originalAlertRuleId"
+                        ] = snapshot_activity.get().previous_alert_rule.id
```

Comment:
You can use `previous_alert_rule_id` here instead, which helps us avoid making a query to fetch `previous_alert_rule` from postgres